docker/test/stateful/run.sh

```diff
@@ -36,10 +36,26 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
 echo "UBSAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 echo "LLVM_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
```

Comment:
You can remove this also as symbolizer paths are set in test base docker image. Stateless and stateful tests use it as common ancestor 
https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/base/Dockerfile#L53-L55

Reply:
Ok.